gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageReadChannel.java

```diff
@@ -995,13 +1031,21 @@ protected InputStream openStream(long bytesToRead) throws IOException {
           cacheFooter(response);
           if (retriesCount != 0) {
             logger.atInfo().log(
-                "Successfully cached footer after %s retries for '%s'", retriesCount, resourceId);
+                "Successfully cached footer context:%d,retries:%s,time:%d,resource:%s",
+                Thread.currentThread().getId(),
```

Comment:
I don't think we need to log the threadId, it should already be part of the logging pattern (thread Id/name)